app/models/generic_object_definition.rb

```diff
@@ -21,6 +23,7 @@ class GenericObjectDefinition < ApplicationRecord
   REG_ATTRIBUTE_NAME = /\A[a-z][a-zA-Z_0-9]*\z/
   REG_METHOD_NAME    = /\A[a-z][a-zA-Z_0-9]*[!?]?\z/
   ALLOWED_ASSOCIATION_TYPES = (MiqReport.reportable_models + %w(GenericObject)).freeze
+  IMPORT_CLASS_NAMES = %w(GenericObjectDefinition).freeze
```

Comment:
Is this used anywhere?

Reply:
@Fryguy yes, to [validate the imported data](https://github.com/romanblanco/manageiq/blob/bz1595259/app/models/mixins/yaml_import_export_mixin.rb#L47)